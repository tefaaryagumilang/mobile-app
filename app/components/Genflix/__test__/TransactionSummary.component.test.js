import React from 'react';
import renderer from 'react-test-renderer';
import Component from '../TransactionSummary.component';

describe('Customizable Transaction Summary component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Component/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
