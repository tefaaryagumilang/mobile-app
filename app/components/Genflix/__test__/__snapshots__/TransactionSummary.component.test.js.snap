// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customizable Transaction Summary component renders correctly 1`] = `
<View
  style={
    Object {
      "paddingVertical": 20,
    }
  }
>
  <View
    style={
      Object {
        "paddingBottom": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "fontFamily": "Roboto",
          "fontWeight": "700",
        }
      }
    >
      Pada Selasa, 13 Jun 2023
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <View>
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": undefined,
              "fontSize": 30,
            },
            Object {
              "color": "#c38a05",
              "marginRight": 10,
            },
            Object {
              "fontFamily": "simas",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
    <View>
      <Text
        style={
          Array [
            Object {
              "fontWeight": "700",
            },
            Object {
              "color": "#000000",
              "fontFamily": "roboto",
            },
          ]
        }
      >
        N/A
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "roboto",
            },
          ]
        }
      >
        N/A
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "roboto",
            },
          ]
        }
      >
        N/A
      </Text>
    </View>
  </View>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 25,
        },
        Object {
          "color": "#ccc",
          "marginBottom": 5,
          "marginLeft": 3,
        },
        Object {
          "fontFamily": "simas",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "paddingBottom": 20,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": undefined,
            "fontSize": 30,
          },
          Object {
            "color": "#0787e3",
            "paddingRight": 10,
            "paddingTop": 10,
          },
          Object {
            "fontFamily": "simas",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      
    </Text>
    <View>
      <Text
        style={
          Array [
            Object {
              "fontWeight": "700",
            },
            Object {
              "color": "#000000",
              "fontFamily": "roboto",
            },
          ]
        }
      >
        N/A
      </Text>
      <Text
        style={
          Object {
            "color": "#000000",
            "fontFamily": "roboto",
          }
        }
      >
        N/A
      </Text>
      <Text
        style={
          Object {
            "color": "#000000",
            "fontFamily": "roboto",
          }
        }
      >
        N/A
      </Text>
    </View>
  </View>
</View>
`;
